src/renderer/features/_shared/classes/DiceRoller.ts

```diff
@@ -38,14 +14,12 @@ class SkillRollResult implements ISkillRollResult {
   }
 
   constructor(
+    total: number,
```

Comment:
because they're getting the `|| 0` sets, should these be optional parameters? That way, the eg rollSkillCheck wouldn't have to stub out the unused params

Reply:
Fair.  Changed for next push.  I had some trouble with sorting out optional parameters early on, but this time I seem to have made the linter stop screaming at me.